### PR TITLE
BE-531 #Done Send outlinks through status stream in JSoupParserBolt

### PR DESCRIPTION
**Pull request by nfalaki**
We want to send outlinks through status stream in JSoupParserBolt

### CODE REVIEW

**Comment by nfalaki** [comment]
@jnioche @GuiForget please review.

File: jsoup-parser/src/main/java/com/shopstyle/crawler/jsoup/JSoupParserBolt.java [hunk 1]
```diff
@@ -188,18 +192,26 @@ public void execute(Tuple tuple) {
...
+        for (String outlink : linksKept) {
+            // configure which metadata gets inherited from parent
+            Metadata linkMetadata = metadataTransfer.getMetaForOutlink(url, metadata);
+            collector.emit(com.digitalpebble.storm.crawler.Constants.StatusStreamName, new Values(
+                    outlink, linkMetadata, Status.DISCOVERED));
+        }
 
-        collector.emit(tuple, new Values(url, content, metadata, text.trim(), linksKept));
+        eventCounter.scope("parsed").incr();
+        collector.emit(new Values(url, content, metadata, text.trim()));
```
Comment (jnioche) [inline jsoup-parser/src/main/java/com/shopstyle/crawler/jsoup/JSoupParserBolt.java]: the outgoing tuple was anchored, it should remain so
```
collector.emit(tuple, ...
```

**Comment by nfalaki** [comment]
@jnioche made the changes

**Comment by nfalaki** [comment]
@GuiForget Julien gave this his thumbs up.

**Pull request merged**